apps/server/src/routes/tags/tag-service.ts

```
@@ -0,0 +1,293 @@
+			.from(tag)
+			.where(and(eq(tag.id, tagId), eq(tag.userId, userId)))
+			.limit(1);
+
+		if (!tagRecord.length) return null;
+
+		const fileCount = await db
+			.select({ count: count() })
+			.from(fileTag)
+			.where(and(eq(fileTag.tagId, tagId), eq(fileTag.userId, userId)));
+
+		const record = tagRecord[0]!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)